client/keys/add.go

```diff
@@ -331,18 +331,22 @@ func runAddCmd(ctx client.Context, cmd *cobra.Command, args []string, inBuf *buf
 	if err != nil {
 		return err
 	}
+	noBackup, _ := cmd.Flags().GetBool(flagNoBackup)
+	showMnemonic := !noBackup
+	showMnemonicDiscreetly, _ := cmd.Flags().GetBool(flagIndiscreet)
```

Comment:
This looks like inverted

Reply:
Yep, I'm working on this...

Reply:
Done

![image](https://github.com/cosmos/cosmos-sdk/assets/25278203/0380f777-9f75-4bcb-b85b-9cf9cd7d5807)